ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -39,21 +39,29 @@ public ClinicalDetailsModel(INotificationService service, NtbsContext context, I
             this.auditService = auditService;
         }
 
-        public override async Task<IActionResult> OnGetAsync(int? id)
+        public override async Task<IActionResult> OnGetAsync(int? id, bool isBeingSubmitted)
         {
             Notification = await service.GetNotificationWithNotificationSitesAsync(id);
             if (Notification == null)
             {
                 return NotFound();
             }
 
-            ClinicalDetails = Notification.ClinicalDetails;
             NotificationId = Notification.NotificationId;
+            NotificationStatus = Notification.NotificationStatus;
+            Notification.SetFullValidation(NotificationStatus, isBeingSubmitted);
```

Comment:
Could we put those two lines into a method in Notificationbase, considering they're called in every razor page?

Reply:
I have tried to commonise whole method with abstract methods but seemed more confusing.
And I agree, I will try to commonise some bits of it now